api/src/main/java/org/pmiops/workbench/api/UserController.java

```diff
@@ -145,31 +148,41 @@ private BillingAccount freeTierBillingAccount() {
 
   @Override
   public ResponseEntity<WorkbenchListBillingAccountsResponse> listBillingAccounts() {
+    final List<BillingAccount> billingAccounts = Lists.newArrayList();
+
+    // if we show the free credits account, it goes first
+
+    if (freeTierBillingService.userHasRemainingFreeTierCredits(userProvider.get())) {
```

Comment:
This is actually not quite the right logic. Free credits are attributed to the workspace creator.  Therefore the right thing to check would be the creator's credits.  This may require a bit more thinking and maybe a quick chat with Karthik to think about how we can handle this cleanly.

i.e. perhaps only the creator can touch the billing settings - OR perhaps only the creator sees the free tier settings (unless a workspace currently free tier, then it should always display)

In all cases, I think there are valid situations where you'd want to show the free tier account in the list despite the user being over the limit, so I think we'll want to not have the logic here.